### DISCUSSION TITLE
ansi escape sequences formatting

### DISCUSSION THREAD

**Discussion by kuerbisbeule33** [allure-framework/allure2#2485] [Questions & Support] [answered]
If I generate the Allure report, I notice that the raw ANSI escape sequences appear in the stdout section of the report (Execution -> Test Body -> Stdout). Consequently, I can see the escape sequences but not their formatting. Is there a way to replicate the same formatting (colors) in the Allure report as seen in the actual console output?
![raw_ansi_escapes](https://github.com/allure-framework/allure2/assets/85802951/50047c29-0a7a-4fa7-add2-5c80501a8d61)

**Comment by delatrie** [comment]
I've created a feature request in the allure-python repository. Here is the issue to track: allure-framework/allure-python#806.

**Comment by sarzamas** [comment]
up!

**Comment by baev** [comment, marked as answer]
Fixed via https://github.com/allure-framework/allure2/pull/2949

**Reply by amezin** [reply]
But... no, it's not: 
![image](https://github.com/user-attachments/assets/860f2d11-ae51-4048-8a44-1c3269c2a654)
